src/interfaces/ICSModule.sol

```diff
@@ -188,85 +189,29 @@ interface ICSModule is IQueueLib, INOAddresses, IAssetRecovererLib {
-    /// @param referrer Optional. Referrer address. Should be passed when Node Operator is created using partners integration
-    function addNodeOperatorETH(
-        uint256 keysCount,
-        bytes memory publicKeys,
-        bytes memory signatures,
-        NodeOperatorManagementProperties memory managementProperties,
-        bytes32[] memory eaProof,
-        address referrer
-    ) external payable;
-
-    /// @notice Add a new Node Operator using stETH as a bond.
-    ///         At least one deposit data and corresponding bond should be provided
-    /// @notice Due to the stETH rounding issue make sure to make approval or sign permit with extra 10 wei to avoid revert
-    /// @param keysCount Signing keys count
-    /// @param publicKeys Public keys to submit
-    /// @param signatures Signatures of `(deposit_message_root, domain)` tuples
-    ///                   https://github.com/ethereum/consensus-specs/blob/v1.4.0/specs/phase0/beacon-chain.md#signingdata
-    /// @param managementProperties Optional. Management properties to be used for the Node Operator.
-    ///                             managerAddress: Used as `managerAddress` for the Node Operator. If not passed `msg.sender` will be used.
-    ///                             rewardAddress: Used as `rewardAddress` for the Node Operator. If not passed `msg.sender` will be used.
-    ///                             extendedManagerPermissions: Flag indicating that managerAddress will be able to change rewardAddress.
-    ///                                                         If set to true `resetNodeOperatorManagerAddress` method will be disabled
-    /// @param permit Optional. Permit to use stETH as bond
-    /// @param eaProof Optional. Merkle proof of the sender being eligible for the Early Adoption
-    /// @param referrer Optional. Referrer address. Should be passed when Node Operator is created using partners integration
-    function addNodeOperatorStETH(
-        uint256 keysCount,
-        bytes memory publicKeys,
-        bytes memory signatures,
-        NodeOperatorManagementProperties memory managementProperties,
-        ICSAccounting.PermitInput memory permit,
-        bytes32[] memory eaProof,
-        address referrer
-    ) external;
-
-    /// @notice Add a new Node Operator using wstETH as a bond.
-    ///         At least one deposit data and corresponding bond should be provided
-    /// @notice Due to the stETH rounding issue make sure to make approval or sign permit with extra 10 wei to avoid revert
-    /// @param keysCount Signing keys count
-    /// @param publicKeys Public keys to submit
-    /// @param signatures Signatures of `(deposit_message_root, domain)` tuples
-    ///                   https://github.com/ethereum/consensus-specs/blob/v1.4.0/specs/phase0/beacon-chain.md#signingdata
+    /// @notice Permissioned method to add a new Node Operator
```

Comment:
would add here role name that can call the method

Reply:
we usually don't add the role name
added an explainer in the comment